src/components/Profile/index.tsx

```diff
@@ -76,6 +77,12 @@ const ViewProfile: NextPage = () => {
           <Details profile={profile as any} />
         </GridItemFour>
         <GridItemEight className="space-y-5">
+          {currentProfile && profile && currentProfile.id !== profile.id && (
+            <NewPost
+              defaultContent={`@${profile?.handle} `}
+              placeholder={`Write a message to @${profile?.handle}`}
```

Comment:
Im wondering what placeholder we should give here. Ideally we are not sending a message to the user rather we are mentioning the user in a post. Should we change it to something else @bigint?

Reply:
Maybe `Mention @handle in a post` ?

Reply:
We have a little problem going on here. You can see that the mention is not actually a mentionNode and suggestions are popping up. May be we should make use of `createMentionNode` and create a mention node. Right now it is a paragraph node?

Reply:
Instead of pulling the logic all the way up here, can't we rather "parse" the default string in `Editor` component and convert the "@mention" string to a `MentionNode` ? It would create a more usable dev experience.

Reply:
We can do that. If we parse the `@mention` to be `MentionNode` we right now won't know if it actually is a valid mention or not while constructing in the editor. random text with `@asdasd` will also be considered as `MentionNode`. Rather we want to keep only valid mentions while creating a post. That is why we have a type ahead menu with suggestions and then convert it to `MentionNode` making sure it is a valid mention while entering it into editor.

Reply:
Right. I will have a look later today if I can create the `MentionNode` directly and update the prop types.

Another way I started thinking about - to simplify the value propagation to `Editor` - isn't there a way to set `EditorState` somehow in lenster custom state?

Reply:
@harish-sethuraman I tried to implement `$createMentionNode` directly in `Profile` component, but there's a problem that `Editor` doens't exists yet.

So I implemented 2nd solution - which parses the `@mention` and creates `TextNode` and `MentionNode` according to the parsed `defaultContent` string. The solution works properly (parsing), but as you mentioned above, it will create `MentionNode` even for strings which are not a correct profile. I'm not sure if that has any side effects for posting messages.

**Screenshot of the problem with missing `Editor`**

<img width="997" alt="Screenshot 2022-11-15 at 16 34 33" src="https://user-images.githubusercontent.com/988706/201965913-74e0868f-106f-4844-83f5-101a9029b93d.png">

Reply:
@harish-sethuraman shall we merge this?

Reply:
@harish-sethuraman thinking about the parsing topic. If/when we have case where you would like to avoid parsing of `@mention` to `MentionnNode` we can implement prop `disableParsing` or find other way how to go around this. However, until we have such a case, I think we can merge this in.

Reply:
Give me some time. I'll look into this in the weekend.

Reply:
@harish-sethuraman are you gonna look @ this 🙇🏼?

Reply:
Have to 😭  not getting time